.github/workflows/ci.yaml

```diff
@@ -21,48 +27,47 @@ jobs:
       fail-fast: false
       matrix:
         version: [3.11]
-        os: [ubuntu-latest]
+        os: [ubuntu-22.04]
```

Comment:
Ugh I'm going to need to change the required check for this...

![image](https://github.com/user-attachments/assets/6094e4c7-f650-408d-a086-b295f0760894)
![image](https://github.com/user-attachments/assets/6bfd6ffb-d0d1-4922-a1dd-0c178999f293)